.github/scripts/ci.sh

```diff
@@ -5,19 +5,41 @@ set -eu -o pipefail
 
 DIR="${PWD}"
 
-curl --silent --show-error --location https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh | bash -s -- --fabric-version "$1"
+# for fabric 2.5 we don't test for BFT
+if [[ $1 == '2.5.3' && $3 == 'BFT' ]]; then
+    exit 0
+fi
 
+curl --silent --show-error --location https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh | bash -s -- --fabric-version "$1"
 cd fabric-samples/test-network
-if [[ $2 == 'create_channel' ]]; then
-    ./network.sh up
-else    
-    ./network.sh up createChannel -c mychannel
+
+echo $1
+echo $3
+
+if [[ $1 == '3.0.0-preview' && $3 == 'BFT' ]]; then 
+    echo start 3.0.0 preview BFT processing
+    if [[ $2 == 'create_channel' ]]; then
+        ./network.sh up -bft
+    else    
+        echo y | ./network.sh up createChannel -bft -c mychannel
+    fi
+else
+    if [[ $2 == 'create_channel' ]]; then
+        ./network.sh up
+    else    
+        ./network.sh up createChannel -c mychannel
+    fi
 fi
 
 cd "${DIR}/test"
 mkdir -p ../organizations
 cp -r ../fabric-samples/test-network/organizations/* ../organizations/
 ls ../organizations/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.crt
+
+if [[ $3 == 'BFT' ]]; then
+    export BFT=true
+fi
```

Comment:
Since the BFT environment variable needs to be set here based on the CONSENSUS parameter, and is used by later e2e test code, maybe it would be simpler if the golang.yaml workflow just set a CONSENSUS environment variable to the matrix.CONSENSUS value?